setup.py

```diff
@@ -221,42 +221,42 @@ def check_package_status(package, min_version):
         }
     ],
     "decomposition": [
-        {"sources": ["_online_lda_fast.pyx"], "include_np": True},
+        {"sources": ["_online_lda_fast.pyx"]},
         {"sources": ["_cdnmf_fast.pyx"], "include_np": True},
     ],
     "ensemble": [
         {"sources": ["_gradient_boosting.pyx"], "include_np": True},
     ],
     "ensemble._hist_gradient_boosting": [
-        {"sources": ["_gradient_boosting.pyx"], "include_np": True},
-        {"sources": ["histogram.pyx"], "include_np": True},
-        {"sources": ["splitting.pyx"], "include_np": True},
-        {"sources": ["_binning.pyx"], "include_np": True},
-        {"sources": ["_predictor.pyx"], "include_np": True},
-        {"sources": ["_bitset.pyx"], "include_np": True},
-        {"sources": ["common.pyx"], "include_np": True},
+        {"sources": ["_gradient_boosting.pyx"]},
+        {"sources": ["histogram.pyx"]},
+        {"sources": ["splitting.pyx"]},
+        {"sources": ["_binning.pyx"]},
+        {"sources": ["_predictor.pyx"]},
+        {"sources": ["_bitset.pyx"]},
+        {"sources": ["common.pyx"]},
     ],
     "feature_extraction": [
         {"sources": ["_hashing_fast.pyx"], "language": "c++", "include_np": True},
     ],
     "linear_model": [
-        {"sources": ["_cd_fast.pyx"], "include_np": True},
-        {"sources": ["_sgd_fast.pyx.tp"], "include_np": True},
-        {"sources": ["_sag_fast.pyx.tp"], "include_np": True},
+        {"sources": ["_cd_fast.pyx"], "include_np": False},
+        {"sources": ["_sgd_fast.pyx.tp"], "include_np": False},
+        {"sources": ["_sag_fast.pyx.tp"], "include_np": False},
```

Comment:
What's the difference between setting `"include_np": False` and omitting it as done above (e.g. l.231)?

Reply:
No difference.